data_models/github_data_model.yml

```
@@ -0,0 +1,16 @@
+AnalysisType: datamodel
+LogTypes: 
+  - GitHub.Audit
+DataModelID: Standard.Github.Audit
+DisplayName: Github Audit
+Filename: github_data_model.py
+Enabled: true
+Mappings:
+  - Name: actor_user
+    Path: actor
+  - Name: assigned_admin_role
+    Path: get_admin_role
```

Should this be `Method`?
```suggestion
  - Name: assigned_admin_role
    Method: get_admin_role
```